react-validated-input-component/dist/styles.css

```diff
@@ -0,0 +1,34 @@
+* {
+  box-sizing: border-box;
+  font-family: 'Roboto', sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  padding: 1rem;
+}
+
+.heading {
+  margin: 0.5rem 0;
+}
+
+.icon {
+  margin: 0 0.5rem;
+}
+
+.fa.fa-check {
+  color: green;
+}
+
+.fa.fa-times {
+  color: red;
+}
```

Comment:
Avoid directly styling 3rd-party classes if you can. Instead, define an additional class to use alongside them. Modifying 3rd-party CSS classes is a bad habit to get into, as it alters the behavior stated in the library's documentation, which confuses teammates.